plugin-server/tests/main/db.test.ts

```diff
@@ -1095,4 +1095,44 @@ describe('DB', () => {
             )
         })
     })
+
+    describe('doesPersonBelongToCohort()', () => {
+        let team: Team
+        let cohort: Cohort
+        let person: Person
+
+        beforeEach(async () => {
+            team = await getFirstTeam(hub)
+            cohort = await hub.db.createCohort({
+                name: 'testCohort',
+                description: '',
+                team_id: team.id,
+                version: 10,
```

Comment:
can we have another test where version is not set, to simulate static cohorts?

Reply:
What is the value then? Null? Something else?

Reply:
Looking at production, seems like version is still set for static cohorts: https://metabase.posthog.net/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjpudWxsLCJuYXRpdmUiOnsicXVlcnkiOiIiLCJ0ZW1wbGF0ZS10YWdzIjp7fX0sInR5cGUiOiJuYXRpdmUifSwiZGlzcGxheSI6InRhYmxlIiwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319

Reply:
refresh the link? Shows up empty for me

Reply:
The query was `SELECT * FROM posthog_cohort WHERE version IS NULL`. But in any case, added the test + fixed handling of nulls. Thanks for pointing this out!